Userland/DevTools/Inspector/main.cpp

```diff
@@ -40,7 +40,6 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     TRY(Core::System::unveil("/res", "r"));
     TRY(Core::System::unveil("/bin", "r"));
     TRY(Core::System::unveil("/tmp", "rwc"));
-    TRY(Core::System::unveil("/sys/kernel/processes", "r"));
```

Comment:
There's no `%sid` parsing here, so are we sure this is safe to remove?

Reply:
Probably not, I will fix this.